awx/ui_next/src/components/JobList/JobList.jsx

```diff
@@ -238,6 +240,19 @@ function JobList({ i18n, defaultParams, showTypeColumn = false }) {
                   onDelete={handleJobDelete}
                   itemsToDelete={selected}
                   pluralizedItemName={i18n._(t`Jobs`)}
+                  cannotDelete={item =>
```

Comment:
Since pluralization can be complicated its best to use this syntax. https://github.com/ansible/awx/blob/705881123c3e8e923ecd45236b989bbdc23449c1/awx/ui_next/src/screens/Inventory/InventoryList/InventoryList.jsx#L222

Once we upgrade Lingui we will adopt `<Plural/>`

Reply:
@AlexSCorey, I saw this syntax. And I attempted to use it. But the string reported during unit-tests was not sane. I will take a better look if I missed something.